sysutils/eksctl/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+maintainers         {@szczad gmail.com:szczad} \
+                    openmaintainer
+
+name                eksctl
+version             0.6.0
+revision            1
+supported_archs     x86_64
+
+categories          sysutils
+platforms           darwin linux
+license             Apache-2.0
+homepage            https://eksctl.io/
+
+description         Simple CLI tool for creating clusterrs on AWS EKS
+long_description    eksctl is a simple CLI tool for creating clusters on EKS - Amazon’s \
+                    new managed Kubernetes service for EC2. It is written in Go, uses \
+                    CloudFormation, was created by Weaveworks.
+
+github.setup        weaveworks eksctl 0.6.0
+github.tarball_from releases
+
+distname            eksctl
+
+platform linux {
+    distfiles       eksctl_Linux_amd64.tar.gz
+    checksums       eksctl_Linux_amd64.tar.gz \
+                    rmd160  ac9754fcdf6d1bda164714503ee0a0bc2ed02499 \
+                    sha256  95d3087c6d84d9451ea68b1fc76e16c7ee57a6e1d644c9e02cdb9e83bdc2ce8c \
+                    size    22121376
+}
+
+platform darwin {
+    distfiles       eksctl_Darwin_amd64.tar.gz
```

Comment:
`distfiles` should never be set directly. It should be properly set by the GitHub portgroup anyway.

Reply:
Until upstream coders fix the source code and the app can be built from the source code, the portfile need to specify which binary to download from Github.